src/sass/messaging/partials/_message-compose.scss

```diff
@@ -50,18 +56,36 @@
 
 .messaging-recipient > div,
 .messaging-from,
-.messaging-subject-group {
+.messaging-subject-field {
   padding: 1.5rem;
+
+  @media (min-width: $medium-screen) {
+    padding-top: .75rem;
+    padding-bottom: .75rem;
+  }
+}
+
+.messaging-subject-field legend {
+  // Should match padding above;
+  @media (max-width: $medium-screen) {
+    padding-top: 1.5rem;
+  }
 }
 
 .messaging-from,
 .messaging-recipient,
-.messaging-subject-group {
+.messaging-subject-field {
   @media (max-width: $medium-screen) {
     border-bottom: 1px solid $color-gray;
   }
 }
 
+[id=messaging-compose] {
```

Comment:
They're the same thing in that they select the same element. But id selectors (e.g. `#messaging-compose`) have higher specificity than attribute selectors (e.g. `[id=messaging-compose]`).

You end up needing a longer selector in order to override those styles -- e.g. `#messaging-compose.variation label`. With the lower-specificity selector we could just use `.variation label` and take advantage of the cascade. 

It's not a huge deal here. There's a low likelihood that we'll need to override these styles. And `#messaging-compose` is slightly shorter. But elsewhere in the Vets.gov CSS, use of id selectors has added a bunch of bloat.